spaceship/lib/spaceship/connect_api/models/app.rb

```
@@ -0,0 +1,508 @@
+      # @return (Bool) Was something changed?
+      def ensure_version!(version_string, platform: nil, client: nil)
+        client ||= Spaceship::ConnectAPI
+        app_store_version = get_edit_app_store_version(client: client, platform: platform)
+
+        if app_store_version
+          if version_string != app_store_version.version_string
+            attributes = { versionString: version_string }
+            app_store_version.update(client: client, attributes: attributes)
+            return true
+          end
+          return false
```

RuboCop (1.89.0)

[convention] Redundant `return` detected.

(Style/RedundantReturn)